nucypher_ops/cli/tbtcv2.py

```diff
@@ -0,0 +1,174 @@
+from nucypher_ops.constants import DEFAULT_NAMESPACE, DEFAULT_NETWORK
+from nucypher_ops.ops.fleet_ops import CloudDeployers
+import os
+import click
+emitter = click
+
+
+@click.group('tbtcv2')
+def cli():
+    """deploy tbtcv2/random beacon nodes"""
+
+
+@cli.command('stage')
+@click.option('--image', help="The docker image to deploy", default='keepnetwork/keep-client:latest')
+@click.option('--namespace', help="Namespace for these nodes.  Used to address hosts and data locally and name hosts on cloud platforms.", type=click.STRING, default=DEFAULT_NAMESPACE)
+@click.option('--network', help="The network name these hosts will run on.", type=click.STRING, default=DEFAULT_NETWORK)
+@click.option('--include-host', 'include_hosts', help="specify hosts to target", multiple=True, type=click.STRING)
+@click.option('--env', '-e', 'envvars', help="Environment variables used during execution (ENVVAR=VALUE)", multiple=True, type=click.STRING, default=[])
+@click.option('--cli', '-c', 'cliargs', help="additional cli launching arguments", multiple=True, type=click.STRING, default=[])
+def stage(image, namespace, network, include_hosts, envvars, cliargs):
+    deployer = CloudDeployers.get_deployer('tbtcv2')(emitter,
+                                                     docker_image=image,
+                                                     namespace=namespace,
+                                                     network=network,
+                                                     envvars=envvars,
+                                                     cliargs=cliargs,
+                                                     resource_name='tbtcv2')
+
+    hostnames = deployer.config['instances'].keys()
+    if include_hosts:
+        hostnames = include_hosts
+    for name, hostdata in [(n, d) for n, d in deployer.config['instances'].items() if n in hostnames]:
```

Comment:
Why?  What happened to my comrade in verbosity and comprehensibility?  

https://www.getyarn.io/yarn-clip/4cfd97ad-d2a1-4142-b776-f438ebf46a88

Reply:
I intend to clean and dry these entry points up, this is just some copypasta. ask @damonC! 😜